### PR TITLE
1834 add option to reveal in file explorer

### LINKED ISSUE #1834: Diagram Can Be Revealed in File Explorer
## __Is your feature request related to a problem? Please describe.__

There is no easy way to get to a diagram file that is open in the Camunda Modeler. The only way to figure out where that file is on your hard drive is the tooltip that pops up on hovering the tab.

image

## __Describe the solution you'd like__

Adding an option to reveal the diagram in the file explorer when opening the context menu would be helpful.

### Current context menu:

image

### _Reveal in file explorer_ option in Visual Studio Code:

image

### PR DESCRIPTION
**Pull request by MaxTru**
closes #1834
Screenshot_20210819_082610

### CODE REVIEW

**Review by nikku — APPROVED** [review]
It works as intended.

Great contribution :tada:

File: app/lib/index.js [hunk 1]
```diff
@@ -186,6 +187,14 @@ renderer.on('dialog:show', async function(options, done) {
   done(null, response);
 });
 
+renderer.on('dialog:open-file-explorer', async function(options, done) {
```
Comment (barmac) [inline app/lib/index.js:190]: No need for the callback to be `async`

Comment (barmac) [reply app/lib/index.js:190]:
```suggestion
renderer.on('dialog:open-file-explorer', function(options, done) {
```

Comment (MaxTru) [reply app/lib/index.js]: :+1: ty, adjusted

**Review by barmac — APPROVED** [review]
I've tested this manually and couldn't break it. Great feature!

**Review by andreasgeier — APPROVED** [review]
✅ &nbsp; LGTM

**Pull request merged**